client/src/actions/user_signup.js

```diff
@@ -0,0 +1,165 @@
+// User Actions
```

Comment:
Also if we're planning on splitting our components into the features they're associated with (Trends, SignUp/LogIn, Entries, etc.) then we should probably split our actions that way as well.